__tests__/integration.test.js

```diff
@@ -238,3 +238,21 @@ describe('/api/comments/:comment_id', () => {
         })
     })
 })
+describe('/api/users', () => {
+    test('GET:200, responds with an array of user objects with the correct properties', () => {
+       return request(app)
+       .get('/api/users')
+       .expect(200)
+       .then(({body}) => {
+        const { users } = body;
+        expect(users.length).toBe(4);
+        users.forEach((user) => {
+            expect(typeof user.username).toBe('string')
+            expect(typeof user.name).toBe('string')
+            expect(typeof user.avatar_url).toBe('string')
+        })
```

Comment:
This looks great 🏅 
Worth checking because I could be misremembering but do the users have some kind of ID key ? ( _check in seed function where users table is created_ ) 
If they do then it would also be good to test for that property too